### ISSUE TITLE
Some business systems still showing QD as their business org, despite QD not being in the org list anymore

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#125]
Example: https://dev.ea.gsa.gov/#/systems/753
Under the business tab, it still has QD for its Business Org, but if you go to the [org chart](https://dev.ea.gsa.gov/#/organizations), there is no QD listed anymore (since FAS removed it in their last reorg).

I think this may be an issue where QD is represented in a separate table.

**Comment by hatfieldjm4** [comment]
It looks like this information is hard-coded (i.e. not referencing the org table, but simply the org name typed in the table itself) into the obj_systems_subsystems table. Ideally, we'd pull the business POC for the business system, cross reference his/her org in the obj_poc list, and that would be the business org.

**Comment by hatfieldjm4** [comment]
In GEAR's dev database, I updated all the QD apps on the obj_systems_subsystems table to reference other orgs in FAS, which I researched using the org to which the business POC (from Archer) belonged. This is not a sustainable way to do this since it is not referencing a lookup table. Not referencing a lookup table will make standardization difficult (e.g. a small difference in the name of one org could make it look like 2 different orgs if you were attempting to see the number of business systems under one 2-letter org, for example). 

I will add another issue to address this at a later date.
